src/java.base/windows/native/libjli/cmdtoargs.c

```diff
@@ -324,6 +324,7 @@ class Vector {
     bool check() {
         // "pgmname" rest of cmdline ie. pgmname + 2 double quotes + space + cmdline from windows
         char* cptr = (char*) malloc(strlen(argv[0]) + sizeof(char) * 3 + strlen(cmdline) + 1);
+        if (cptr == NULL) return false;
```

Comment:
This doesn't look right, I think it has no choice but to print an error and doabort..

Reply:
Hi Alan, I adjusted this part of the coding.